framework/src/base/MooseApp.C

```diff
@@ -609,6 +613,74 @@ MooseApp::MooseApp(InputParameters parameters)
                     name(),
                     " to remove this deprecation warning.");
 
+  // register capabilities
```

Comment:
Can we move this into a method so that we don't bloat the constructor even more?